src/mbgl/gl/index_buffer.hpp

```diff
@@ -14,6 +14,10 @@ class IndexVector {
 public:
     static constexpr std::size_t groupSize = DrawMode::bufferGroupSize;
 
+    void reserve(std::size_t size) {
+        v.reserve(size);
```

Comment:
I see what you meant earlier now - I did not properly took into account the templates parameter pack done via the custom `emplace_back`, thanks 🙇